frontend/packages/console-dynamic-plugin-sdk/src/webpack/ConsoleRemotePlugin.ts

```diff
@@ -52,6 +52,13 @@ export class ConsoleRemotePlugin {
       compiler.options.output.enabledLibraryTypes.push(remoteEntryLibraryType);
     }
 
+    compiler.options.plugins.push(
+      new webpack.IgnorePlugin({
+        resourceRegExp: /.*/,
+        contextRegExp: /node_modules\/@openshift-console\/dynamic-plugin-sdk\/lib\/api/,
```

Comment:
We should configure the `IgnorePlugin` in the following way:
- `resourceRegExp` (requested module specifier) - should start with `@console/`
- `contextRegExp` (resolved module context path) - we don't need this filter, do we?

Reply:
updated with 
```
resourceRegExp: /^@console\//,
contextRegExp: /node_modules\/@openshift-console\/dynamic-plugin-sdk\//,
```